internal/services/pubkey_service.go

```diff
@@ -94,6 +129,12 @@ func DeletePubkey(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = DeleteResource(w, r, id)
```

Comment:
Each time I need to get different dao, if I am not mistaken. I split it in two functions so it is a bit more readable. The nesting in the for loop was a bit scary but there was no way to prevent that, so I decided to do at least this.
But if you think it looks clear even in one function, then that is great, I can merge it into one function. :)